node/request.js

```diff
@@ -110,6 +111,31 @@ TChannelRequest.prototype.hookupCallback = function hookupCallback(callback) {
     function onError(err) {
         if (called) return;
         called = true;
+
+        if (err.isErrorFrame) {
+            self.channel.emitStat(new Stat.Counter(
+                'outbound.calls.systems-errors', 1, {
+                    'target-service': self.serviceName,
+                    'service': self.headers.cn,
+                    // TODO should always be buffer
+                    'target-endpoint': String(self.arg1),
+                    'error-type': err.codeName
```

Comment:
Recommending "caller", "callee", "method".

This would look like the following with StatEmitter:

```
// Channel
self.outboundCallsSystemErrors = self.defineCounter('outbound.calls.system-errors')
// Request
self.channel.outboundCallsSystemErrors.increment(1, {…})
```